slack-api-model/src/main/java/com/slack/api/model/admin/App.java

```diff
@@ -12,6 +12,7 @@ public class App {
     private String privacyPolicyUrl;
     private String appHomepageUrl;
     private String appDirectoryUrl;
+    @SerializedName("is_granular_bot_app")
```

Comment:
```suggestion
    @SerializedName("is_granular_bot_app")
    private boolean granularBotApp
```

Reply:
If this flag can be absent in some patterns, `private Boolean isGranularBotApp` without SerializedName annotation is suitable.